test/core/core_convert.cpp

```diff
@@ -31,9 +31,9 @@ namespace
 
 bool convert_rgb32f_rgb9e5(const char* FilenameSrc, const char* FilenameDst)
 {
-	if(FilenameDst == NULL)
+	if(FilenameDst == nullptr)
 		return false;
-	if(std::strstr(FilenameDst, ".dds") > 0 || std::strstr(FilenameDst, ".ktx") > 0)
+	if(std::strstr(FilenameDst, ".dds") == nullptr && std::strstr(FilenameDst, ".ktx") == nullptr)
```

Comment:
Is this supposed to verify that `FilenameDst` contains `.dds` or `.ktx`? Before it was returning false if it contained either, now it returns false if it contains neither. Let me know if this isn't the intended behaviour and I'll change it back.